### PR TITLE
Add missing awaits in Launch_Latiss*.ipynb. Add extra exp definition …

### PR DESCRIPTION
**Pull request by isotuela**
…and include a print statement in QFM method in PSF_evaluation.ipynb

### CODE REVIEW

File: procedures/auxtel/reduction_procedures/PSF_evaluation.ipynb [hunk 1]
```diff
@@ -71,7 +71,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "dataId = {'day_obs': 20220505, 'seq_num': 902, 'detector': 0, \"instrument\": 'LATISS'}"
+    "dataId = {'day_obs': 20220914, 'seq_num': 750, 'detector': 0, \"instrument\": 'LATISS'}"
```
Comment (patrickingraham) [inline procedures/auxtel/reduction_procedures/PSF_evaluation.ipynb]: This should be a frame that is in the test dataset... just so it always runs.
I believe the following should work:
` "dataId = {'day_obs': 20200315, 'seq_num': 120, 'detector': 0, \"instrument\": 'LATISS'}" `

**Review by patrickingraham — CHANGES_REQUESTED** [review]
Looks good, just one minor issue to be fixed.

File: procedures/auxtel/reduction_procedures/PSF_evaluation.ipynb (hunk 1 above)
Comment (isotuela) [reply procedures/auxtel/reduction_procedures/PSF_evaluation.ipynb]: Done

**Review by patrickingraham — APPROVED** [review]
This now looks correct. Thank you.

**Pull request merged**